### ISSUE TITLE
[FR] Multi-select or single select SwiftEntry

### ISSUE THREAD

**Issue by serjooo** [huri000/SwiftEntryKit#98]
Hello! 

Based on a need I thought of implementing a SwiftEntry preset which allows you to select or multi-select items in a UITableView or UICollectionView. 

The way I thought about it is that we can allow the developer to define their custom Cells or use a default cell, that has a title and accessory image with its selected accessory image. Thus, the developer would just need to pass models implementing this requirement. 

As much as it would be desirable to make this with a UITableView, I think with a UICollectionView you can achieve much more different layouts with this single SwiftEntry preset.

Moreover, I guess we would need to define the maximum size the SwiftEntry can expand to before the UITableView or UICollectionView content becomes scrollable instead of expanding as much as it can to fit its content on the screen.

Would love to hear your thoughts about this and how you would go about implementing this feature!

ps: Thank you for this great project!

**Comment by SyedIsmailAhamed** [comment]
I was trying to display a UIViewcontroller with a UITableView using SwiftEntry but my table view delegate methods(didSelectRow) are not getting called, same UIViewController when tried through the UIPopOverController works perfectly fine.

**Comment by serjooo** [comment]
@huri000 Hello! I hope I'm not bothering you, waiting on your feedback to start working on this feature if possible. If you don't see the feature fitting the project at the moment, I could just implement it on my Fork without merging it with the main project. Thank you again.

**Comment by huri000** [comment]
Hi @serjooo, thanks a lot for your suggestion!
To be honest, I think that's a bit out of scope in terms of customization of SwiftEntryKit. 
Also, collection views are very customizable in their nature - every developer has his own preferences as for the looks and feels of his collection view based feature.
Therefore, I prefer this feature to be shelved for now.
Please let me know your thoughts.

**Comment by huri000** [comment]
@SyedIsmailAhamed - Please read the README section about User Interaction. I'm pretty sure you need to define touch absorbance for the entry.

**Comment by serjooo** [comment]
Thanks for your reply! Yes I do agree with you. Especially that SwiftEntryKit does allow to display ViewControllers as entries; however, I was thinking of abstracting the whole selection process and keeping track of the selection, or making a specific view/view controller each time you want a multi-select entry. Moreover, we could allow the collection view to be configured as the user pleases by passing the datasource and layout to it, and dequeing whatever cell or cells they wish, but I guess also that would mean that you still would have a lot to do to implement the SwiftEntry, so no major benefit is won.

**Comment by ios3** [comment]
Use this attributes.screenInteraction = .forward

**Comment by idevChandra6** [comment]
> Use this attributes.screenInteraction = .forward

I have this problem too. `.forward` does not work.
Is there any other way that make `UITableView` respond to touches when presented as entry (`UIViewController` has `UITableView`)
